docs/source/workflow_inputs.rst

```diff
@@ -411,10 +411,11 @@ Modeling a whole SFA/MF building is defined in ``/HPXML/SoftwareInfo/extension``
 
 For these simulations:
 
+- An HPXML file with multiple ``Building`` elements is used, where each ``Building`` represents an individual dwelling unit.
```

Comment:
Of course, we all recognize that it's confusing. But it's not so easy to address given that HPXML was mostly created without MF buildings in mind. And if we did address it, it would be a breaking change for every HPXML user. So we've tried to minimize changes to the schema and live with it.

Reply:
You can't just rename it to `Unit` as there are HPXML users that describe the whole MF building using that element (e.g., 50,000 sqft, 20 units, 60 bedrooms, etc.).